### DISCUSSION TITLE
socket for Arduino

### DISCUSSION THREAD

**Discussion by bestbean** [kachurovskiy/nanoels#53] [General]
Recently, i destroyed 2 arduinos, one by connecting 7.5V power source to 5V arduino pin, other by placing a completely build and powered on board over the last solder leftover on the table. omg.
Now i'll build the arduino on plug sockets.
OLED display heigth could be adequately adapted. 
Could the case stl adapted for this?

**Comment by kachurovskiy** [comment]
It happens, I stopped counting the number of parts I went through before got some sense on handling them. Nano costs just 2-3$ I guess but the sad thing are all the other parts on the board which must be trashed and re-soldered on the new PCB because it's almost impossible to cleanly remove the damaged Nano.

Adjusting the case would be hard, it's always several iterations and I don't have the target board to test it. However if you tell me by how many millimeters you want to raise the front panel, I can give it a try.

I also shared the Fusion file in https://github.com/kachurovskiy/nanoels/blob/main/h1/case/nanoels-box.f3d in case you want to give it a spin. Cheers!

P.S.: There's a reason they send min. 5 PCBs in the order ;-)

**Comment by bestbean** [comment]
> Nano costs just 2-3$
Last year, i found Arduinos for this price. But now, i only found much more expensive.

**Reply by kachurovskiy** [reply]
Hmm, you're right, they're actually 6 euro both on AliExpress and Amazon - https://www.amazon.de/gp/product/B078SBBST6

**Reply by bestbean** [reply]
a Raspberry Pi Mikrocontroller PICO would be cheaper.
133 MHz ARM Cortex-M0+ Dual-Core  2x I2C  2x SPI 16x PWM ...

**Reply by kachurovskiy** [reply]
Not really, 8.85 with delivery :) Not to mention that in some places you just can't get it. But I admit that Nano is by no means the best possible microcontroller to use - just a well-known working one with good library support.

**Reply by bestbean** [reply]
Meanwhile, i gave 3 nanoELS boards away, together with 3 nanos. One working nano here to destroy ...

**Reply by kachurovskiy** [reply]
Wow, you're a kind soul. I could just mail you something (PCB / assembled NanoEls) if you're in Europe and at the wits end with this.

**Reply by bestbean** [reply]
Thanks! Maybe i come back to your offer after destroying my last arduino ...
5 new boards are on their way to me.
